portfolio/src/main/webapp/task.js

```diff
@@ -6,20 +6,129 @@
  * converted into a JSON string.
  */
 class Task {
-  // TODO(raulcruise): could you add the type for each param please? 
-  
+  // TODO(raulcruise): could you add the type for each param please?
+
   /**
    * All fields are required except description, which will be null
    * when not set.
    * @param name: name of task as String
    * @param description: description of task as String, can be null
-   * @param duration: duration of task as Duration class
-   * @param priority: priority level of task, range 1-5 inclusive. 
+   * @param duration: duration of task
```

Comment:
What is the type of duration? What is the time unit?

Reply:
Done.